packages/ui/package.json

```diff
@@ -1,6 +1,6 @@
 {
     "name": "@ferlab/ui",
-    "version": "10.4.6",
+    "version": "10.4.7-rc1",
```

Comment:
je pense que tu as pas fait le npm install car le packag-lock.json devrait aussi avoir la version mise à jour

Reply:
en effet, fixing it now